libheif/heif.cc

```diff
@@ -458,6 +458,44 @@ void heif_context_debug_dump_boxes_to_file(struct heif_context* ctx, int fd)
   (void) written;
 }
 
+
+//Created by Devon Sookhoo
+LIBHEIF_API
+void heif_add_box_example(char* input_filename) {
+
+  //CREATE CONTEXT 
```

Comment:
minor personal style preference: I don't like all caps unless it is in the name of a global constant